next.config.mjs

```diff
@@ -35,6 +35,26 @@ const nextConfig = {
           "https://join.slack.com/t/prairielearn/shared_invite/zt-13kx0hg6v-uuC3kyt_3iBxjSpyhCbYVw",
         permanent: true,
       },
+      {
+        source: "/oer",
+        destination: "/catalog/oer",
+        permanent: true,
+      },
+      {
+        source: "/gallery/questions",
+        destination: "/catalog/questions",
```

Comment:
This (and the one for `/gallery/assessments` below) don't correctly handle paths with this as their prefix. That is, we'd want `/gallery/questions/checkbox` to redirect to `/catalog/questions/checkbox`, but it currently doesn't do that. See the docs for "Wildcard path matching" here: https://nextjs.org/docs/app/api-reference/next-config-js/redirects#wildcard-path-matching

Reply:
similar comment as above for the question page. We will no longer have pages in `gallery/questions`, so can we just ignore these changes for the questions path, and just update them for the assessments?

Reply:
well, I updated both just in case.

Reply:
We can address that in #73, but what we should do there is redirect anything like `/gallery/questions/:slug*` to `/gallery/questions` so that the user doesn't get a 404.

Reply:
but we are removing `/gallery/questions`, so should we redirect to an existing page? so maybe I should just do `/gallery/questions/:slug*` to `/catalog/questions/`?

Reply:
That was I typo, I updated my comment.